fmriprep/workflows/bold/base.py

```diff
@@ -341,10 +346,21 @@ def init_func_preproc_wf(bold_file, ignore, freesurfer,
         name='bold_confounds_wf')
     bold_confounds_wf.get_node('inputnode').inputs.t1_transform_flags = [False]
 
+    # Apply transforms in 1 shot
+    # Only use uncompressed output if AROMA is to be run
+    bold_bold_trans_wf = init_bold_preproc_trans_wf(
+        mem_gb=mem_gb['resampled'],
+        omp_nthreads=omp_nthreads,
+        use_compression=not (low_mem and use_aroma),
```

Comment:
Why is `use_aroma` here? The AROMA workflow does not use the BOLD-space series, does it? I think you probably just want `not low_mem`.

Reply:
@chrisfilo is there any contraindication of aroma with compression?

Reply:
I suspect this is a holdover from `bold_mni_trans_wf`. The reasoning there would be that ICA-AROMA will be able to benefit from mmapping, but if we're not going to run ICA-AROMA then the output isn't used except as a derivative, so we might as well go ahead and compress.

If I understand the confounds situation now, the T1w-resampled BOLD series are only used for derivatives, so we want:

* `bold_bold_trans_wf`: `use_compression=not low_mem`
* `bold_reg_wf`: `use_compression=False`
* `bold_mni_trans_wf`: `use_compression=not (low_mem and use_aroma)`

Also, I've noticed that resampling to T1w is not contingent on T1w derivatives. If there's no T1w output then we should probably drop this, to save the time and space of creating those files.

Reply:
Let's fix it in a dedicated PR -> #878